pyrealm/pmodel/new_subdaily.py

```diff
@@ -181,15 +207,15 @@ def __init__(
 
         # 5) Calculate the realised daily values from the instantaneous optimal values
         self.xi_real: NDArray = memory_effect(
-            self.pmodel_acclim.optchi.xi, alpha=alpha, handle_nan=handle_nan
+            self.pmodel_acclim.optchi.xi, alpha=alpha, allow_holdover=allow_holdover
```

Comment:
This form seems to be used more widely, so not something to address in this PR. If we'd decide to change it that should be a new issue.

Reply:
Not ideal because it is unclear when things are a variable or argument name? I can see that. I think this occurs naturally when an option is being passed through to an internal function. Here we have `SubdailyModel(..., allow_holdover = True, ...)` which is directly used in all `memory_effect` calls so it seems natural to use the same names to make it clear which arguments are being passed down?

Reply:
I was never taught programming 😄 (Well, technically, I was but it was [BBC Basic](https://en.wikipedia.org/wiki/BBC_BASIC)...). I can see the value in that - but as you say it is a different PR.